FirstSolution/Part10Task6/Task.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace Part10Task6
+{
+   
+    // Преобразовать 4ый метод, чтобы он ничего не возвращал,
+    // если years == 0 (используя оператор return)
+    
+    public class Task
+    {
+        private static void Main(string[] args)
+        {
+            PrintTextDependingOnAge("Ilya", 0);
+        }
+
+        private static string PrintTextDependingOnAge(string name, int age)
+        {
+            if (age > 0)
+            {
+                if (age == 1)
+                {
+                    return String.Format("My name is {0} and I'm {1} year old", name, age);
+                }
+                else
+                {
+                    return String.Format("My name is {0} and I'm {1} years old", name, age);
```

Comment:
You don't use returned value in calling method

Reply:
Changed